store/mongo/migrations.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/mendersoftware/go-lib-micro/log"
 	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
 	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 )
 
```

Comment:
`DbVersion = "2.0.0"` is this correct?

Reply:
That is indeed an error. I bumped it to 2.0.1.